packages/server/src/modules/tag/defaults.ts

```diff
@@ -14,9 +14,11 @@ const defaultTags = [
   'Mate',
 ].map(text => ({ text, id: service.generateIndex() }));
 
-TagModel.findOne({ text: defaultTags[0].text }).then(result => {
-  if (result) {
-    return;
-  }
-  TagModel.insertMany(defaultTags);
-});
+console.warn('Move to migration');
```

Comment:
You can undo this changes to avoid conflicts. I've moved it to migrations in the open PR.